docs/integrations/saas-cloud/1password.md

```diff
@@ -155,15 +167,35 @@ _sourceCategory="app/"
 | fields -_messageTime
 ```
 
-## Collecting logs for 1Password
+## Collection configuration and app installation
+
+Depending on the set up collection method, you can configure and install the app in three ways:
+
+- **[Create a new collector and install the app](#create-a-new-collector-and-install-the-app)**. Create a new Sumo Logic Cloud-to-Cloud (C2C) source under a new Sumo Logic Collector and later install the app; Or
+- **[Use an existing collector and install the app](#use-an-existing-collector-and-install-the-app)**. Create a new Sumo Logic Cloud-to-Cloud (C2C) source under an existing Sumo Logic Collector and later install the app; Or
+- **[Use existing source and install the app](#use-an-existing-source-and-install-the-app)**. Use your existing configured Sumo Logic Cloud-to-Cloud (C2C) source and install the app.
+
```

Comment:
If you use this exact wording in multiple articles, you probably should make a reuse file for it.

Reply:
Sure, John!